frontend/src/main/components/RiderApplication/RiderApplicationEditForm.js

```diff
@@ -1,54 +1,75 @@
-import React from 'react'
+import React from 'react';
 import { Button, Form } from 'react-bootstrap';
-import { useForm } from 'react-hook-form'
+import { useForm } from 'react-hook-form';
 import { useNavigate } from 'react-router-dom';
+import { useBackendMutation } from 'main/utils/useBackend';
 
-function RiderApplicationEditForm({ initialContents, submitAction, email}) {
+function RiderApplicationEditForm({ initialContents, submitAction, email }) {
     const navigate = useNavigate();
-    
+
     // Stryker disable all
     const {
         register,
         formState: { errors },
         handleSubmit,
         getValues
-    } = useForm(
-        { defaultValues: initialContents }
-    );
+    } = useForm({ defaultValues: initialContents });
     // Stryker enable all
```

Comment:
`// Stryker enable all` should be changed to `// Stryker restore all`.

`// Stryker enable all` doesn't actually do anything, so all of the Stryker checking is disabled after this line.

Reply:
In UsersTable.js, which I didn't touch, there are multiple // Stryker enable all comments. Are these mistakes in the code? I imagine if they were to be replaced by // Stryker restore all, many mutations would survive UsersTable.test.js.

<img width="477" alt="Screenshot 2024-05-22 at 5 34 44 PM" src="https://github.com/ucsb-cs156-s24/proj-gauchoride-s24-5pm-7/assets/114462302/c7965492-ad85-4be8-8701-0875b7f832d6">